CHANGELOG.next.asciidoc

```diff
@@ -388,6 +388,11 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 *Functionbeat*
 
+- Add basic ECS categorization and `cloud` fields. {pull}19174[19174]
```

Comment:
Seems like there are too many lines in here but assuming the PR goes green, I would do a follow up PR to get rid of these lines to not have to rerun CI.